lib/scout/widgets/filter_menu_error.dart

```diff
@@ -1,16 +1,27 @@
+import 'package:ax_dapp/wallet/bloc/wallet_bloc.dart';
 import 'package:flutter/material.dart';
 
 class FilterMenuError extends StatelessWidget {
-  const FilterMenuError({super.key});
+  const FilterMenuError({
+    this.selectedChain = EthereumChain.sxMainnet,
+    super.key,
+  });
+  final EthereumChain selectedChain;
 
   @override
   Widget build(BuildContext context) {
-    return const SizedBox(
-      height: 70,
-      width: 400,
+    final warningText = selectedChain == EthereumChain.polygonMainnet
+        ? 'Change to SX network for NFL Tokens'
+        : selectedChain == EthereumChain.sxMainnet
+            ? 'Change to Polygon network for MLB Tokens'
+            : 'Athletes not supported yet';
+
+    return SizedBox(
+      height: 80,
+      width: 600,
       child: Text(
-        'Athletes not supported yet',
-        style: TextStyle(color: Colors.red, fontSize: 30),
+        warningText,
+        style: TextStyle(color: Colors.amber, fontSize: 30),
```

Comment:
we need to make this TextStyle object as const variable
ex: const TextStyle(color: Colors.amber, fontSzie: 30)